h2o-docs/src/product/performance-and-prediction.rst

```diff
@@ -2747,6 +2750,166 @@ Retrieving graphs via R is not yet supported. An `.ipynb demo showing this examp
         shap.summary_plot(shap_values, X, plot_type="bar")
 
 
+Interventional / Marginal SHAP
```

Comment:
@hannah-tillman would you be so kind and have a quick look at the documentation, just in case something is not clear and needs rephrasing and/or typos etc.? Thanks in advance!